knowhere/settings.py

```diff
@@ -114,7 +114,7 @@
         'ENGINE': 'django.db.backends.mysql',
         'NAME': 'knowhere',
         'USER': 'knowhere_system',
-        'PASSWORD': '!catjetpacks123',
+        'PASSWORD': '!catjets1234',
```

Comment:
Yeah these slipped through in the commit. I'm not sure how my db got saved
with a different password but I've just been lazy about changing it.
On Nov 18, 2015 9:13 AM, "Ben" <notifications@github.com> wrote:

> In knowhere/settings.py
> <https://github.com/justKnoWhere/knowhere/pull/37#discussion_r45210872>:
>
> > @@ -114,7 +114,7 @@
> >          'ENGINE': 'django.db.backends.mysql',
> >          'NAME': 'knowhere',
> >          'USER': 'knowhere_system',
> > -        'PASSWORD': '!catjetpacks123',
> > +        'PASSWORD': '!catjets1234',
>
> Why change the password?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/justKnoWhere/knowhere/pull/37/files#r45210872>.
>